source/rpt/parameters_rpt.cc

```diff
@@ -202,7 +200,56 @@ Parameters::DetectorParameters::parse_parameters(ParameterHandler &prm)
     radius                  = prm.get_double("radius");
     length                  = prm.get_double("length");
     detector_positions_file = prm.get("detector positions file");
+
```

Comment:
I don't know if this part needs more comments. Looks good to me.

Reply:
I would maybe add a quick comment here on the format of the file.
Like just quickly recap in a comment what are the headers to be expected. The more this is documented, the better it is.